module/VuFind/src/VuFind/Role/Assertion/DeveloperPermissionAssertion.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * Developer Permission Assertion
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2025.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Authorization
+ * @author   Juha Luoma <juha.luoma@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     http://vufind.org/wiki/ Wiki
+ */
+
+namespace VuFind\Role\Assertion;
+
+use LmcRbacMvc\Service\AuthorizationService;
+use VuFind\Db\Entity\UserEntityInterface;
+
+/**
+ * Developer Permission Assertion
+ *
+ * @category VuFind
+ * @package  Authorization
+ * @author   Juha Luoma <juha.luoma@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     http://vufind.org/wiki/ Wiki
+ */
+class DeveloperPermissionAssertion implements \LmcRbacMvc\Assertion\AssertionInterface
```

Comment:
Would it be better to rename this to `HasVerifiedEmailAssertion` so it's more reusable?